### PR TITLE
ResizeObserver

### PR DESCRIPTION
**Pull request by larsjohnsen**
The API is only available for Chrome under a flag (chrome://flags/#enable-experimental-web-platform-features), and isn't yet implemented for the other browsers. It does however perform significantly better than the current solution.

### CODE REVIEW

**Comment by larsjohnsen** [comment]
Chrome 64+ has ResizeObserver enabled by default.

File: lib/modules/showImages.js [hunk 1]
```diff
@@ -1818,73 +1791,45 @@ function makeMediaMovable(media, element, dragInitiater = element) {
...
-			});
-		}
-
-		window.addEventListener('resize', debouncedResize);
-
-		if (prevExpand) return prevExpand();
+	const updateWrapper = ({ width, height }) => {
+		wrapper.style.height = `${horizontalIsVertical ? width : height}px`;
+		// Force immediate reflow in order to get resizeObserver to react faster
+		window._trash_updateWrapperRefowProvocation = wrapper.getBoundingClientRect();
```
Comment (erikdesjardins) [inline lib/modules/showImages.js]: Debugging?

Comment (larsjohnsen) [reply lib/modules/showImages.js]: This made the whole process go a bit faster, but that's not the seemingly not case anymore.

**Comment by larsjohnsen** [comment]
@erikdesjardins Were you able to get `resize-observer-polyfill` working? It apparently breaks when run through `transform-dead-code-elimination`. Even after that plugin is removed, it for some reason does not trigger. `resize-observer-lite` however works fine as it simply wraps `element-resize-detector` when ResizeObserver is not available.

**Comment by erikdesjardins** [comment]
Hmm, yeah, I thought it was working at some point but apparently not.

**Comment by erikdesjardins** [comment]
Test failure is spurious

**Pull request merged**